OfekTwitter/css/normalize.css

```diff
@@ -1,3 +1,4 @@
+/* ok */
```

Comment:
delete all conments, both auto-generated, and lines explainations.
no need to explain simple lines, like `margin: 0;`